core/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -198,7 +198,7 @@ public Translog(
                 logger.debug("wipe translog location - creating new translog");
                 Files.createDirectories(location);
                 final long generation = 1;
-                Checkpoint checkpoint = new Checkpoint(0, 0, generation, globalCheckpointSupplier.getAsLong());
+                Checkpoint checkpoint = new Checkpoint(0, 0, generation, Long.MAX_VALUE, Long.MIN_VALUE, globalCheckpointSupplier.getAsLong());
```

Comment:
Negative values for translog seem to have a special meaning. Maybe this should be `0, 0`?